src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/apiUsage/IsEmptyFunctionUsageInspector.java

```diff
@@ -120,27 +118,14 @@ private boolean isNullableCoreType(@NotNull Set<String> resolvedTypesSet) {
                     return false;
                 }
 
-                return  resolvedTypesSet.contains(Types.strInteger) ||
-                        resolvedTypesSet.contains(Types.strFloat)   ||
-                        resolvedTypesSet.contains(Types.strBoolean) ||
-                        resolvedTypesSet.contains(Types.strResource);
+                return resolvedTypesSet.contains(Types.strInteger) ||
+                       resolvedTypesSet.contains(Types.strFloat) ||
+                       resolvedTypesSet.contains(Types.strBoolean) ||
+                       resolvedTypesSet.contains(Types.strResource);
             }
         };
     }
 
-    public JComponent createOptionsPanel() {
-        return OptionsComponent.create((component) -> {
-            component.addCheckbox("Report empty() usage", REPORT_EMPTY_USAGE, (isSelected) -> REPORT_EMPTY_USAGE = isSelected);
-            component.addCheckbox("Suggest to use count()-comparison", SUGGEST_TO_USE_COUNT_CHECK, (isSelected) -> SUGGEST_TO_USE_COUNT_CHECK = isSelected);
-            component.addCheckbox("Suggest to use null-comparison", SUGGEST_TO_USE_NULL_COMPARISON, (isSelected) -> SUGGEST_TO_USE_NULL_COMPARISON = isSelected);
```

Comment:
@rentalhost : are those settings gone completely, or you've relocated them somewhere?

Reply:
Done. I have checked other inspections and seems that it was the unique issue (uff!).